pkgs/by-name/nh/nh/package.nix

```
@@ -7,14 +7,14 @@
   fetchFromGitHub,
   nix-update-script,
   nvd,
-  nix-output-monitor,
+  # Make optional as nom is AGPL. This mirrors nix-community/nh/package.nix
+  use-nom ? true,
+  nix-output-monitor ? null,
   buildPackages,
 }:
+assert use-nom -> nix-output-monitor != null;
```

I am not sure why this is set to null?
```suggestion
  use-nom ? true,
  nix-output-monitor,
  buildPackages,
}:
```

just this would not leave nom in the closure if use-nom is false.